src/pages/i18n/index.tsx

```diff
@@ -167,15 +175,20 @@ export const I18N = (): JSX.Element => {
             </Toolbar>
 
             <List id={'item-list'}>
-                {Object.keys(fruits).map((fruit, index) => (
+                {fruits.map((fruit, index) => (
                     <InfoListItem
                         key={index}
-                        onClick={(): void => selectFruit(fruit)}
+                        onClick={(): void => selectFruit(fruit.name)}
                         ripple={true}
                         style={{ textAlign: isRTL() ? 'right' : 'left' }}
-                        title={t(`FRUITS.${fruit}`)}
-                        subtitle={t('MORE_INFO')}
-                        icon={<Checkbox checked={selectedItems.has(fruit)} onChange={(): void => selectFruit(fruit)} />}
+                        title={t(`FRUITS.${fruit.name}`)}
+                        subtitle={t('CURRENCY', { price: fruit.price, lng: lang })}
```

Comment:
I think you can skip passing in the lang here since the t function should already know it. I did a quick test locally and it seemed to still be working correctly.

Reply:
That's pretty cool, yep it works without explicitly passing in the language.